src/components/Search.vue

```diff
@@ -2,24 +2,43 @@
   <div class="d-flex flex-items-center">
     <Icon name="search" size="22" class="mb-1 mr-2 text-gray" />
     <input
-      :value="value"
+      :value="searchValue"
       :placeholder="placeholder"
       @input="handleInput"
-      type="text"
+      type="search"
       autocorrect="off"
       autocapitalize="none"
-      class="border-0 input flex-auto"
+      class="border-0 input flex-auto width-full"
+    />
+    <Icon
+      v-if="searchValue"
+      name="close"
+      size="16"
+      class="mb-1 text-gray button-close"
+      @click="clearInput"
     />
   </div>
 </template>
 
 <script>
 export default {
-  props: ['value', 'placeholder'],
+  props: ['searchValue', 'placeholder'],
   methods: {
     handleInput(e) {
-      this.$emit('input', e.target.value);
+      this.$emit('update:searchValue', e.target.value);
+    },
+    clearInput() {
+      this.$emit('update:searchValue', '');
     }
   }
 };
 </script>
+
+<style scoped lang="scss">
+.button-close {
```

Comment:
Can you remove this style and keep the same style we use on proposal creation here:
![image](https://user-images.githubusercontent.com/16245250/95213062-98291a80-0818-11eb-9474-5e83d2f88869.png)